src/main/scala/uk/sky/fs2/kafka/topicloader/TopicLoader.scala

```diff
@@ -97,15 +100,21 @@ trait TopicLoader {
 
   private def filterBelowHighestOffset[F[_] : Monad : LoggerFactory, K, V](
       logOffsets: NonEmptyMap[TopicPartition, LogOffsets]
-  ): Pipe[F, ConsumerRecord[K, V], ConsumerRecord[K, V]] = {
+  ): Pipe[F, ConsumerRecord[K, V], ConsumerRecord[K, V]] = stream => {
+    val logger = LoggerFactory[F].getLogger
+
     val nonEmptyOffsets =
       logOffsets.toSortedMap.filter((_, o) => o.highest > o.lowest)
 
     val allHighestOffsets: HighestOffsetsWithRecord[K, V] =
       HighestOffsetsWithRecord[K, V](nonEmptyOffsets.map((p, o) => p -> (o.highest - 1)))
 
-    _.evalScan(allHighestOffsets)(emitRecordRemovingConsumedPartition[F, K, V])
+    stream
+      .scan(allHighestOffsets)(emitRecordRemovingConsumedPartition[K, V])
       .takeWhile(_.partitionOffsets.nonEmpty, takeFailure = true)
+      .evalTapChunk(_.partitionLast.traverse { last =>
+        logger.warn(s"Finished loading data from ${last.topicPartition.show} at offset ${last.offset}")
```

Comment:
i know this hasn't changed but why is this a warning? this should be info

Reply:
[`66541f3` (#64)](https://github.com/sky-uk/fs2-kafka-topic-loader/pull/64/commits/66541f33f1b6dc5f73651a4431b936348d251438)

Was like that in the Akka lib, always assumed it was intentional 👀 I've lowered it to info

Reply:
https://github.com/sky-uk/kafka-topic-loader/blob/0f6c46d858e9d56287eca159eec4a3dc3924ae6c/src/main/scala/uk/sky/kafka/topicloader/TopicLoader.scala#L249

Reply:
I stand corrected, guess I left it on while debugging and forgot to lower it 😨